### PR TITLE
V1 setup repo and publish initial version of module, with tests

### PR DESCRIPTION
**Pull request by bdgamble**

### CODE REVIEW

**Comment by bdgamble** [comment]
@antonbazhal small tweaks from your module to make it more generic, let me know if you have any feedback

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
-takes lambda event and context and allows publishing to a specified dlq
+
+Module that sends a lambda event, along with error details and lambda context info to the provided deadletter queue url via AWS SQS.
+
+# Usage
+
+## new Publisher(dlqUrl, logger)
+Constructor for the class.
+- **dlqUrl** - { String, required } - valid url to the deadletter queue you wish to send a message to.
+- **logger** - { Object/Function, optional } - a valid logger (minimum must have info and error functions), or a function that takes the lambda context and returns a logger. If no logger is provided, defaults to using `console`.
```
Comment (AntonBazhal) [inline README.md]: Maybe, it's better not to log at all if there is no logger provided?

**Comment by AntonBazhal** [comment]
Future improvement: SQS has message size limit, so messages that are bigger than limit need to be stored in S3 while passing reference to SQS. It'd be nice to support that. To make things easier, maybe it's reasonable to make constructor take options object, where you'd be able to specify additional parameters like S3 bucket, max message size, etc?

File: README.md (hunk 1 above)
Comment (bdgamble) [reply README.md]: yah

**Comment by bdgamble** [comment]
I'll create an issue on the repo to add this support when I get some cycles to loop back on this

**Comment by AntonBazhal** [comment]
Isn't it reasonable to change constructor now, so you do not need to make breaking changes in the future?

**Comment by bdgamble** [comment]
sure, though it wouldn't be a breaking change as it is just adding an additional param to constructor which adds additional functionality, if I add the param now, I should like add the additional functionality, unless you were thinking of putting logger in the options object as well?

**Comment by AntonBazhal** [comment]
yeah, that what I was thinking about

**Comment by bdgamble** [comment]
ok, I will do that change then

**Review by AntonBazhal — APPROVED** [review]
LGTM

**Pull request merged**